docs/user_guide/examples.rst

```diff
@@ -153,7 +153,7 @@ ANTS
 
 
 CAT12
----
+-----
```

Comment:
just got the heading to render properly in the doc because at the moment we get this:

![image](https://github.com/ReproNim/neurodocker/assets/6961185/a4768843-c2e8-4ee5-851a-5878c952736d)